Packs/DeveloperTools/Integrations/CustomIndicatorDemo/IndicatorFields/param2.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
should be under `Packs/DeveloperTools/IndicatorFields/param2.json`

Reply:
Fixed